src/resources/views/crud/fields/rate.blade.php

```diff
@@ -0,0 +1,67 @@
+<!-- radio -->
+@php
+    $value = old($field['name']) ? old($field['name']) : (isset($field['value']) ? $field['value'] : (isset($field['default']) ? $field['default'] : 0 ));
+    $field['count'] = $field['count'] ?? 5;
+@endphp
+
+@include('crud::fields.inc.wrapper_start')
+    <div>
+        <label>{!! $field['label'] !!}</label>
+        @include('crud::fields.inc.translatable_icon')
+    </div>
+
+    <div class="rate form-check">
+        @while ($field['count']--)
+            <input @include('crud::fields.inc.attributes') type="radio" id="star{{ $field['count'] }}" name="{{$field['name']}}" value="{{ $field['count']+1 }}" {{ $field['count']+1 == $value ? 'checked' : '' }}/>
```

Comment:
If the field is used multiple times on a form this line should be like shown bellow.
Thnx @tabacitu, your "[comment ](https://github.com/Laravel-Backpack/CRUD/pull/2676/commits/bbdb60be32920796fe74e078914029c9e213da00#r411191734)" caught my attention, I will extract this field as a package and fix this in the weekend.
```suggestion
            <input @include('crud::fields.inc.attributes') type="radio" id="star-{{ $field['name']-$field['count'] }}" name="{{$field['name']}}" value="{{ $field['count']+1 }}" {{ $field['count']+1 == $value ? 'checked' : '' }}/>
```